classes/apiclient.js

```diff
@@ -435,7 +435,7 @@ class APIClient {
         const { client, config } = this;
         const timeout =
             config.timeout ||
-            (config.mode === 'full' ? 125 * 60000 : 5 * 60000);
+            (config.mode === 'deep'  ? 125 * 60000 : 5 * 60000);
         const initialDelay =
```

Comment:
@danielrea I see that you renamed the full to deep, however I don't the additional code to cover the `standard` analyses. Am I missing something?